nitk-thesis.csl

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>NITK Thesis</title>
+    <title-short>NITK</title-short>
+    <id>http://www.zotero.org/styles/nitk-thesis</id>
+    <link href="http://www.zotero.org/styles/nitk-thesis" rel="self"/>
+    <link href="http://nitk.ac.in/sites/default/files/public/curriculum/PG-Curriculum2016-revised.pdf" rel="documentation"/>
+    <author>
+      <name>Pragadeesh Sekar</name>
+      <email>pragadeeshsekar@gmail.com</email>
+    </author>
+    <contributor>
+      <name>Bruce D'Arcus</name>
+    </contributor>
+    <contributor>
+      <name>Curtis M. Humphrey</name>
+    </contributor>
+    <contributor>
+      <name>Richard Karnesky</name>
+      <email>karnesky+zotero@gmail.com</email>
+      <uri>http://arc.nucapt.northwestern.edu/Richard_Karnesky</uri>
+    </contributor>
+    <contributor>
+      <name>Sebastian Karcher</name>
+    </contributor>
+    <contributor>
+      <name>Pragadeesh Sekar</name>
+      <uri>http://www.mendeley.com/profiles/pragadeesh-sekar/</uri>
+      <email>pragadeeshsekar@gmail.com</email>
+    </contributor>
```

Comment:
Please remove all contributors except yourself. And either list yourself as a contributor or as an author, but not both. (see also https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)